Procfile

```diff
@@ -0,0 +1 @@
+web: node ./bin/www
```

Comment:
Eventually this won't be required as we would push a built image to their Docker registry.